src/lib/components/Codelab/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Element that renders codelab expandable/collapsible codelab
+ * instructions and an embedded Glitch iframe.
+ */
+
+import {html} from "lit-element";
+import {BaseElement} from "../BaseElement";
+
+/** @const {string} */
+const GLITCH_ORIGIN = "https://glitch.com";
+/** @const {string} */
+const GLITCH_EMBED = `${GLITCH_ORIGIN}/embed/#!/embed`;
+
+/**
+ * Render codelab instructions and Glitch
+ * @extends {BaseElement}
+ * @final
+ */
+class Codelab extends BaseElement {
+  static get properties() {
+    return {
+      // Name of the glitch to render in the iframe.
+      glitch: {type: String},
+      // The file to show when the Glitch renders.
+      path: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.glitch = "";
+    this.path = "index.html";
+  }
+
+  createRenderRoot() {
+    const container = document.createElement("div");
```

Comment:
Why do this instead of just having `<div class="web-codelab__glitch">` returned as part of `render()`?

Reply:
The element has some existing Light DOM children so it goes:

```
<web-codelab>
  <div class="web-codelab__instructions">[this gets generated by 11ty]</div>
  .. [this is where web-codelab__glitch needs to render]
</web-codelab>
```

If I try to return `web-codelab__glitch` it will replace the `web-codelab__instructions` child. This works around that by appending a new child and using that as the render target.

Reply:
That's fine but it might be worth just adding a comment here calling this out.